open-reac/src/main/java/com/powsybl/openreac/parameters/input/OpenReacParameters.java

```diff
@@ -48,6 +48,10 @@ public class OpenReacParameters {
 
     private OpenReacSolverLogLevel logLevelSolver = OpenReacSolverLogLevel.EVERYTHING;
 
+    private static final String DEBUG_DIR_KEY = "debug_dir";
+
+    private String debugDir = null;
```

Comment:
```suggestion
    private String debugDir;
```
I don't think we need to set null since default value of String is already null